### ISSUE TITLE
When adding a new interval it would be helpful if it reuses the last interval

### ISSUE THREAD

**Issue by OliverEberlei** [sadekd/nova-opening-hours-field#24]
Hi. Thank you for this great package. 

A small improvement idea: When I enter business hours, I often find myself entering the same interval 5 times (because the location has the same hours from monday-friday for example). It would be a huge timesaver if, instead of adding a random open/close hour every time, it would re-use the last interval I have entered before. Maybe the last one that was entered in updateInterval() ?

Thank you for your consideration. I could also try to do a PR if you're interested? Though I never did this before :)

**Comment by desaintflorent** [comment]
I agree it's time consuming. Or at least be able to copy and past between fields ?
